.builders/scripts/repair_wheels.py

```diff
@@ -97,6 +100,25 @@ def __str__(self):
         ]) + '.whl'
 
 
+def check_unacceptable_files(
+    wheel: Path,
+    bypass_prefixes: list[str],
+    invalid_file_patterns: list[str],
+):
+    """Check if a wheel contains any unacceptable files and exit if found."""
+    if any(wheel.name.startswith(pkg_prefix) for pkg_prefix in bypass_prefixes):
+        return
+
+    unacceptable_files = find_patterns_in_wheel(wheel, invalid_file_patterns)
+    if unacceptable_files:
+        print(
+            f"Found copies of unacceptable files in external wheel '{wheel.name}'",
+            f'(matching {invalid_file_patterns}): ',
+            unacceptable_files,
+        )
+        sys.exit(1)
```

Comment:
Minor (pythonic) suggestion:
```suggestion
        exit(
            f"Found copies of unacceptable files in external wheel '{wheel.name}'",
            f'(matching {invalid_file_patterns}): ',
            unacceptable_files,
        )
```